problemsets-mat223/modules/module12-exercises.tex

```diff
@@ -1,17 +1,41 @@
 \begin{exercises}
+    %Patrick:
+        % 60min
 	\begin{problist}
-		\prob  Example problem with parts
+		\prob  Indicate whether the following linear transformations are surjective and/or injective, as well as indicate whether they are invertible. Justify your answers.
 		\begin{enumerate}
-			\item   The first part
-			\item   The second part
+			\item   $\mathcal S:\R^2\to\R^2$, a linear transformation that doubles every vector.
+			\item   $\mathcal R:\R^2\to\R^2$, a linear transformation that rotates every vector clockwise by $72^\circ$.
+			\item   $\mathcal P:\R^2\to\R^2$, a linear transformation that projects every vector onto the $y$-axis.
+			\item   $\mathcal F:\R^2\to\R^2$, a linear transformation that reflects every vector over the line $y=x$.
+			\item   $T:\R^3\to\R^3$, a linear transformation induced by the matrix $M_T = \mat{1&2&3\\3&4&5\\5&6&7}$.
+			\item   $U:\R^3\to\R^2$, a linear transformation induced by the matrix $M_U = \mat{1&2&3\\3&4&5}$.
 		\end{enumerate}
-		\begin{solution}
-		Solutions to the parts
-			\begin{enumerate}
-				\item   $(2,4,8)$
-				\item   $(15,6,-21)$
-			\end{enumerate}
-		\end{solution}
-
+		
+		\prob Solve the system $\systeme{2x+y=5,3x+7y=3}$ by:
+		    \begin{enumerate}
+		        \item   substitution and elimination methods learned in high school.
+		        \item   finding the inverse matrix of $\mat{2&1\\3&7}$.
+		    \end{enumerate}
+		
+		\prob Solve the system $\systeme{2x+2y+3z=4,2x+2y+z=0, 4x+5y+6z=2}$ by:
+		    \begin{enumerate}
+		        \item   substitution and elimination methods learned in high school.
+		        \item   finding the inverse matrix of $\mat{2&2&3\\2&2&1\\4&5&6}$.
+		    \end{enumerate}
+		    
+		\prob 	Let $A=\mat{1&3&5\\0&2&0\\-1&0&2}$. 
```

Comment:
This should come earlier. There should be a 5-part question that is just "find the inverses, if possible, of the following matrices". 4 of them should be very easy to compute and 1 should be difficult.

Reply:
Ok, I add another problem at the beginning. I want to keep the question by the way.